test/integration/com/unifina/service/ProductServiceIntegrationSpec.groovy

```diff
@@ -61,9 +61,9 @@ class ProductServiceIntegrationSpec extends IntegrationSpec {
 		[s1, s2, s3, s4].eachWithIndex { Stream s, int i -> s.id = "stream-id-${i+1}" }
 		[s1, s2, s3, s4]*.save(failOnError: true, validate: true)
 
-		troll = new SecUser(username: "sylvester", name: "sylvester stallone", password: "x", email: "s@s.com", timezone: "Europe/Helsinki")
+		troll = new SecUser(username: "sylvester", name: "sylvester stallone", password: "x", email: "s@s.com", lastLogin: new Date(0))
```

Comment:
Can remove `lastLogin` if you set the default field value in `SecUser.groovy` (also elsewhere in this file)